### DISCUSSION TITLE
[Idea]: Meeting Countdown

### DISCUSSION THREAD

**Discussion by bucmado** [sircharlo/meeting-media-manager#2453] [Ideas] [answered]
### Is your idea related to a problem?

N/A

### Describe your idea

The Meeting Countdown is a timer that can be configured to automatically show on your media display before the start of a meeting. It shows the minutes and seconds remaining before the meeting should start. The Options, Settings page, Countdown section can be used to configure some aspects of the timer. When turned on, the default settings show a 5-minute countdown in the centre of the display, accompanied by a graphical dial depicting the remaining time as shown below:

Meeting Countdown options

1) Activate countdown:
   - Off
   - On

2) Show Graph:
   - Off
   - On

3) Countdown Duration:
   - 5 minutes
   - 4 minutes
   - 3 minutes
   - 2 minutes
   - 1 minute

4) Size of the countdown:
   - Small
   - Medium
   - Large

5) Position of the countdown:
   - Center top
   - Top left
   - Top right
   - Center
   - Left
   - Right
   - Center bottom
   - Bottom left
   - Bottom right

This organization makes it easier for users to select options as they are logically and clearly grouped.

### Describe alternatives you've considered

N/A

### Additional context

![Screenshot_12kjghhj](https://github.com/sircharlo/meeting-media-manager/assets/56500218/2dd7b8b0-0b37-47e7-8332-3a91fe36325f)
![Screenshot_12rergsgthrtherth](https://github.com/sircharlo/meeting-media-manager/assets/56500218/596af96d-9761-4174-adec-86a56d6aa4cf)

**Comment by e-Feitosa** [comment]
This feature already exists, however, you need to configure it according to your need. I use a 2-minute video with the countdown. And you need to add as recurring media, see the topic where it was commented... https://github.com/sircharlo/meeting-media-manager/discussions/2355

**Comment by sircharlo** [comment, marked as answer]
As @e-Feitosa mentioned, the comments https://github.com/sircharlo/meeting-media-manager/discussions/2355#discussioncomment-8436749 and https://github.com/sircharlo/meeting-media-manager/discussions/2355#discussioncomment-8440561 explain the reasons why we won't be exploring this feature for now.
